### PR TITLE
simplesamlphp/saml2 SSPSA 201612-01: Incorrect signature validation

### PR DESCRIPTION
**Pull request by marcomenzel**

### CODE REVIEW

**Comment by fabpot** [comment]
Looks like the validator found some issues.

File: simplesamlphp/saml2/201612-01.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+title:     Incorrect signature verification
+link:      https://simplesamlphp.org/security/201612-01
+branches:
+    1.9.x:
+        time:     2016-11-29 13:12:00
+        versions: ['1.9.0', '<1.9.1']
```
Comment (fabpot) [inline simplesamlphp/saml2/201612-01.yaml]: should be `>=1.9.0`

**Comment by fabpot** [comment]
Also, can you rebase to remove the merge commit?

File: simplesamlphp/saml2/201612-01.yaml (hunk 1 above)
Comment (marcomenzel) [inline simplesamlphp/saml2/201612-01.yaml]: Amended my commit about five seconds later but you are a way to quick ;-)

**Comment by marcomenzel** [comment]
time to improve my phpstorm git client knowledge :| 
please just use #182 for now

**Pull request closed without merging**